cmd/minishift/cmd/start.go

```diff
@@ -236,9 +236,13 @@ func handleProxies() *util.ProxyConfig {
 	return proxyConfig
 }
 
+// getSlice return slice for provided key if value type is []interface{} otherwise nil
 func getSlice(key string) []string {
 	if viper.IsSet(key) {
-		return viper.GetStringSlice(key)
+		value := viper.Get(key)
+		if _, ok := value.([]interface{}); ok {
```

Comment:
Can you give me a quick run down on what's going on here and why? We did verify this, didn't we? Are we not able to write some tests for this? Some generic test which are centred around getting string slices asserting that no matter of whether we get the slice from the command line or from the config, we end up with the right data. The test would also serve as "documentation" on what's actually expected.